src/components/icon/list.tsx

```diff
@@ -1108,6 +1113,125 @@ export function getIcon(
           />
         </svg>
       );
+    case IconName.polygonUp:
+      return (
+        <svg
+          width="6"
+          height="4"
+          viewBox="0 0 6 4"
```

Comment:
this seems quite small
and kinda aligned to the top because of its size, not center-aligned
is that intended?

Reply:
these sizes are not important since we are setting the SVG width manually in getIcon and GetIcon, and the height will scale automatically.